sdk/cosmos/azure-cosmos/src/test/java/com/azure/cosmos/CosmosDatabaseTest.java

```diff
@@ -55,7 +55,7 @@ public void afterClass() {
     @Test(groups = {"emulator"}, timeOut = TIMEOUT)
     public void createDatabase_withPropertiesAndOptions() throws CosmosClientException {
         CosmosDatabaseProperties databaseDefinition = new CosmosDatabaseProperties(CosmosDatabaseForTest.generateId());
-        databases.add(databaseDefinition.getId());
+        databases.add(ModelBridgeInternal.getResource(databaseDefinition).getId());
```

Comment:
once `getId()` is added back we should undo this change as getId will be required method in CosmosDatabaseProperties.

here and in other places for the properties classes.

Reply:
removed.